src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -128,6 +141,14 @@ public void hasEntry_entryNotInEntryBook_returnsFalse() {
     public void hasEntry_entryInEntryBook_returnsTrue() {
         modelManager.addListEntry(ALICE, Optional.empty());
         assertTrue(modelManager.hasEntry(ALICE));
```

Comment:
We should check using `hasListEntry` instead. We don't do duplicate checks on add entry methods in `Model`, TIL

Reply:
Also `hasEntry` does an OR check with `hasArchivesEntry` and `hasListEntry`. Should we test these things separately?

Reply:
Yup, I've split all the tests and checked that every predicate returns the correct value for each case.